CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ Yarn now accepts sponsorships! Please give a look at our [OpenCollective](https:
 ### Bugfixes
 
 - Yarn now has a proper [governance model](https://github.com/yarnpkg/berry/blob/master/GOVERNANCE.md).
+- Usage of `pnpify` inside directories containing spaces is now possible.
```

Comment:
Should be under the master section as this wasn't released in 3.0.0

Reply:
Ok thanks! Won’t have time to move it now. Please feel free to do so or I’ll have a look at it next week!

Reply:
Alright, moved it